test/helpers/PageTitleSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Could you please move this file to views.helpers as the corresponding class for this test is in views.helpers?